crates/pop-cli/src/commands/call/chain.rs

```diff
@@ -67,10 +70,17 @@ impl CallChainCommand {
 	/// Executes the command.
 	pub(crate) async fn execute(mut self) -> Result<()> {
 		let mut cli = cli::Cli;
+		cli.intro("Call a chain")?;
 		// Check if all fields are specified via the command line.
 		let prompt_to_repeat_call = self.requires_user_input();
 		// Configure the chain.
-		let chain = self.configure_chain(&mut cli).await?;
+		let chain = configure_chain(
```

Comment:
Moved `configure_chain` to a common module for better reusability. The logic is now shared in `pop up`